.gitignore

```diff
@@ -1,6 +1,54 @@
-target/
+/target/
+.gradle
+build/
+!**/src/main/**/target/
+!**/src/test/**/target/
+!**/src/main/**/build/
+!**/src/test/**/build/
 
-# The following files are generated/updated by Hilla
+.DS_Store
+
+# The following files are automatically generated/updated
```

Comment:
Where does all this come from? Who is automatically generating and updating them? Why should we ignore all IDE settings, there even are IDE settings in the repo?